SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Activate Rust in order to recognize the uv command
+PATH="/home/$USER/.local/bin:$PATH"
```

Comment:
Instead of doing this for every type of env, can this be factored out in the form of a variable

```bash
uv = "/home/$USER/.local/bin/uv"
```
that is defined in makenv.sh?

Reply:
Or alternatively, see my comment https://github.com/swan-cern/jupyter-images/pull/138/commits/c1077d15abcbfdf56acc272c08251e7ebe5329ad#r1842190017

Reply:
The only exception is mamba. But I think doing this is fine.